cpp/velox/substrait/VeloxSubstraitSignature.cc

```diff
@@ -168,6 +168,20 @@ TypePtr VeloxSubstraitSignature::fromSubstraitSignature(const std::string& signa
     }
     return std::make_shared<RowType>(std::move(names), std::move(types));
   }
+
+  if (startWith(signature, "list")) {
+    auto listStart = signature.find_first_of('<');
+    auto listEnd = signature.find_first_of('>');
```

Comment:
Do we need to consider nested array? For nested array, `find_first_of('>')` will return wrong ending position for outer array, right?

Reply:
Yes, we should use `find_last_of`.